float64 duration

---

bool success